atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/OpenTransaction.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.transaction.api;
+
+import com.palantir.atlasdb.metrics.Timed;
+
+public interface OpenTransaction extends Transaction {
```

Comment:
Interested in another name, but this is how we call this in atlasdb-proxy.

Reply:
I think this is fine, enough of us work on atlasdb-proxy that the naming should be obvious :)